test/integration/puppetlabs/services/master/master_service_test.clj

```diff
@@ -281,3 +283,76 @@
               (test-path! "hostcert" (str ca-files-test-runtime-dir "/certs/localhost.pem"))
               (test-path! "serial" (str ca-files-test-runtime-dir "/certs/serial"))
               (test-path! "cert_inventory" (str ca-files-test-runtime-dir "/inventory.txt")))))))))))
+
+(def graphite-enabled-config
+  {:metrics {:server-id "localhost"
+             :reporters {:graphite {:update-interval-seconds 5000
+                                    :port 10001
+                                    :host "foo.localdomain"}}
+             :registries {:puppetserver
+                          {:reporters {:graphite {:enabled true}}}}}
+   :jruby-puppet {:master-code-dir test-resources-code-dir
+                  :master-conf-dir master-service-test-runtime-dir}})
+
+(defn get-puppetserver-registry-context
+      [app]
+      (-> app
+          (tk-app/get-service :MetricsService)
+          tk-services/service-context
+          :registries
+          deref
+          :puppetserver))
+
+(defn get-puppetserver-graphite-reporter
+      [app]
+      (:graphite-reporter (get-puppetserver-registry-context app)))
+
+(deftest graphite-filtering-works
+  (testing "default filter works"
+    (let [reported-metrics-atom (atom {})]
+      (with-redefs [metrics-core/build-graphite-sender
+                    (fn [_ domain]
+                      (metrics-testutils/make-graphite-sender reported-metrics-atom domain))]
+        (logutils/with-test-logging
+          (testutils/with-config-dirs
```

Comment:
We may need not the extra `with-puppet-config-files` wrapper here and in the next test.  The `with-config-dirs` wrapper in the [namespace-wide test fixture](https://github.com/puppetlabs/puppetserver/blob/09c359c6343af06e5b7e9b667e94f61d54b82ce5/test/integration/puppetlabs/services/master/master_service_test.clj#L29-L34) would seem to setup the proper files for the tests to run.  At least, it worked for me locally - tests passed - without these extra wrappers.